packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1705,8 +1705,20 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 return { ...attemptData, error };
             }
 
+            const dataStoreTree = this.disableIsolatedChannels ? summarizeResult.summary :
```

Comment:
Can you please add comments explaining what is happening here (in this entire code block you added).